Add CFC VID-1 transcription and summarized topics

pradhansh opened the pull request.

pradhansh committed:
Add CFC VID-1 transcription and summarized topics

Niftalemm reviewed (changes requested):
Seems good. Can u just put the trascription files in a gitginore? The folder was there just for local use and not to be pushed

pradhansh committed:
Ignore Whisper transcription and metadata outputs

pradhansh committed:
Add transcript upload endpoint + metadata support

Added POST /api/videos/{slug}/transcript for uploading transcripts
and minor metadata JSON handling for /api/videos.
